TaskList/TaskList/TaskListHelper.cs

```diff
@@ -0,0 +1,231 @@
+
+        public void FilterBy()
+        {
+            Console.WriteLine($"Input how u want sort \n1. By Priority \n2. By Date");
+
+            switch (Program.ParseInt(Console.ReadLine()))
+            {
+                case 1:
+                    FilterPlansByPriority();
+                    break;
+                case 2:
+                    FilterPlansByDate();
+                    break;
+            }
+        }
+
+        public void EditPlans()
+        {
+            Console.WriteLine("Input task number which u want edit");
+            int inputId = Program.ParseInt(Console.ReadLine());
+
+            if (inputId > count)
+            {
+                Console.WriteLine("Error. number task is no found");
+                return;
+            }
+
+            Console.WriteLine("Edit task:");
+            var inputTaskDate = Console.ReadLine();
+            var task = ParseNewTask(inputTaskDate);
+            taskList[inputId - 1] = task;
+        }
+
+        public void SavePlansTxt()
+        {
+            File.Delete("task.txt");
+            for (int i = 0; i < count; i++)
+            {
+                TaskList task = taskList[i];
+                RegularTask taskDate = new RegularTask();
+
+                try
+                {
+                    using (StreamWriter writeText = new StreamWriter("task.txt", true, System.Text.Encoding.Default))
+                    {
+                        writeText.WriteLine(task?.TaskConvert(i));
+                    }
+                }
+
+                catch
+                {
+                    Console.WriteLine("Write error.");
+                }
+            }
+            Console.WriteLine("Write Successful");
+        }
+
+        public void ReadPlansTxt()
+        {
+            try
+            {
+                using (StreamReader read = new StreamReader("task.txt"))
+                {
+                    Console.WriteLine(read.ReadToEnd());
+                }
+            }
+
+            catch
+            {
+                Console.WriteLine("Error");
+            }
+        }
+
+        public void ViewPlans()
+        {
+            for (int i = 0; i < count; i++)
+            {
+                ColorDisplay(6);
+                TaskList task = taskList[i];
+                RegularTask taskDate = new RegularTask();
+                task?.GetAlarm(taskDate.Date);
+                Console.WriteLine(task?.TaskConvert(i));
+                ColorDisplay(6);
+            }
+        }
+
+        public void AddNewTask()
+        {
+            Console.WriteLine("Input you task name; task date; task time; task priority;");
+
+            if (count > 19)
+            {
+                Console.WriteLine("Error. Task limit is over");
+                return;
+            }
+
+            var inputDate = Console.ReadLine();
+            var task = ParseNewTask(inputDate);
+            WriteTask(task);
+        }
+
+        public TaskList ParseNewTask(string input)
```

Comment:
do not forget to make everything, that is not used from outside, private
```suggestion
        private TaskList ParseNewTask(string input)
```